testgrid/specs/deploy.yaml

```diff
@@ -59,8 +59,14 @@
       version: 19.03.x
     weave:
       version: latest
-  unsupportedOSIDs:
-    - ubuntu-2204 # docker 19.x is not available on ubuntu 22.04
```

Comment:
this unsupported OSIDs section should not have been removed if we're keeping the docker 19.03.x test